README.md

```diff
@@ -10,45 +10,45 @@ Znaczy [pull requestów](https://github.com/nurkiewicz/polski-w-it/pulls), nie t
 
 | Angielski            | Polskawy            | Polski                                |
 | :------------------- | :------------------ | :------------------------------------ |
-| approve              | zapruwować          | zaakceptować                          |
+| approve              | zaapruwować         | zaakceptować                          |
 | authentication       | autentykacja        | uwierzytelnienie (!)                  |
 | authorization        |                     | autoryzacja                           |
 | build                | bild                | budowanie                             |
-| button               | baton               | przycisk                              |
-| cancel               | kanselować          | anulować                              |
+| button               | baton               | przycisk, guzik                       |
+| cancel               | kanselować, anulacja| anulować                              |
 | case                 | kejs                | przypadek                             |
 | confirm              | konfirmować         | potwierdzić                           |
-| content              |                     | zawartość, treść                      |
+| content              | kontent             | zawartość, treść                      |
 | custom               | kastomowy           | specjalny, nietypowy, niestandardowy  |
 | default              | difoltowo           | domyślnie                             |
 | deploy               | zdiplojować         | wdrożyć                               |
-| edge case            |                     | przypadek brzegowy                    |
-| event                |                     | zdarzenie                             |
-| feature              | ficzer              | funkcja                               |
-| feedback             |                     | informacja zwrotna                    |
+| edge case            | edż kejs            | przypadek brzegowy                    |
+| event                | iwent               | zdarzenie, wydarzenie                 |
+| feature              | funkcjonalność      | funkcja, pot. ficzer                  |
```

Comment:
:-1: z dwu powodów, 1. moje rozumienie podlinkowanego wpisu w SJP sugeruje, że 'funkcjonalność' powinna być w innej kolumnie, 2. bardzo mnie mierzi 'ficzer'. Ale jakby zamienić te dwa miejscami...

Reply:
„Posiada pewne funkcjonalności” pojawia się w pytaniu do poradni językowej, nie w odpowiedzi: 
„Funkcjonalność to nie to samo, co funkcja. Funkcjonalność jest nazwą pewnej cechy i w standardowej polszczyźnie pozostaje rzeczownikiem niepoliczalnym, nie występującym normalnie w liczbie mnogiej”.
Co do ficzera, zwróć uwagę na pot. (potocznie).

Reply:
tak też będzie ok:
| feature              | funkcjonalność, ficzer      | funkcja

Reply:
Dzięki za wyjaśnienie, masz rację z "posiada funkcjonalności"!

Ficzer - tak, widziałem. Nie pomogło na reakcję, ale doceniam, że miałeś od początku! :-)

Funkcjonalność - https://sjp.pwn.pl/poradnia/haslo/Funkcja-a-funkcjonalnosc;18108.html, myślę, że tu tkwi kruczek. Takie było moje rozumienie, gdy sugerowałem zamianę kolumn dla _funkcjonalności_.  
_Mogę za wiele wyczytywać_, ale to najnowsza interpretacja i mówi o różnicy dla terminu dla przeciętnego użytkownika (to samo co funkcja przecież) i dla programisty (funkcja coś zwraca, ma parametry, itp.). Dla mnie to rozróżnienie jest wygodne, określa na jakim poziomie rozmawiam (co nie oznacza, że nigdy nie użyję funkcja zamiast funkcjonalność, zwłaszcza tłumacząc coś niekoniecznie technicznemu klientowi).

Natomiast jeśli np. weźmiemy to jako następny PR, to z mojego punktu widzenia można będzie resztę scalić. Albo po prostu poczekać na inne głosy.

Reply:
Ja od razu pomyślałem o tej nieszczęsnej funkcjonalności i już biegłem z prośbą o pociągnięcie :) - moim zdaniem jest to niepoprawna kalka językowa (jak _technology_ - technologia). Funkcjonalny znaczy "dobrze spełniający swoją funkcję". 

W naszej kompomowie funkcjonalność używamy praktycznie zawsze jako "funkcja" albo odpowiednik "ficzura".

angielski | polskawy | polski
 ------- | -------- | --------- 
 functionality | funkcjonalność | funkcja